trio/_core/_run.py

```diff
@@ -1643,10 +1768,10 @@ async def python_wrapper(orig_coro):
             self.instruments.call("task_spawned", task)
         # Special case: normally next_send should be an Outcome, but for the
         # very first send we have to send a literal unboxed None.
-        self.reschedule(task, None)
+        self.reschedule(task, None)  # type: ignore[arg-type]
         return task
 
-    def task_exited(self, task, outcome):
+    def task_exited(self, task: Task, outcome: Outcome[Any]) -> None:
```

Comment:
I .... think the `Any` might possibly be `object`-ified.

Reply:
Unfortunately no, `Outcome` can't be invariant because it uses them in both `unwrap()`'s return value and `send()`'s parameter.